.github/workflows/chatbot.yaml

```diff
@@ -22,6 +24,7 @@ env:
 
 jobs:
   build-and-push-image:
+    if: "!contains(github.event.pull_request.labels.*.name, 'hold-tests')"
```

Comment:
Not sure folks would remember to use this trick

Reply:
I was going to have a discussion with the team about it.  I know I would use it because sometime I just want to see the diff for review purposes and dont want to waist the resources. If your worried about visibility maybe we could tag every PR with `hold-tests` to begin with and then use a `/ok-to-test` similar to prow?